metadata.json

```diff
@@ -1 +1,5 @@
6420746f2065786973742062656c6f772074686174207468726573686f6c642e385472616e736665724661696c65640c250120506572666f726d696e672074686520726571756573746564207472616e73666572206661696c656420666f72206120726561736f6e206f726967696e6174696e6720696e2074686531012063686f73656e2063757272656e637920696d706c656d656e746174696f6e206f66207468652072756e74696d652e204d6f73742070726f6261626c79207468652062616c616e63652069738c20746f6f206c6f77206f72206c6f636b732061726520706c61636564206f6e2069742e4c4d617843616c6c44657074685265616368656408250120506572666f726d696e6720612063616c6c207761732064656e6965642062656361757365207468652063616c6c696e67206465707468207265616368656420746865206c696d697498206f6620776861742069732073706563696669656420696e20746865207363686564756c652e2c4e6f7443616c6c61626c650831012054686520636f6e74726163742074686174207761732063616c6c656420697320656974686572206e6f20636f6e747261637420617420616c6c20286120706c61696e206163636f756e74294c206f72206973206120746f6d6273746f6e652e30436f6465546f6f4c617267650455012054686520636f646520737570706c69656420746f20607075745f636f646560206578636565647320746865206c696d69742073706563696669656420696e207468652063757272656e74207363686564756c652e30436f64654e6f74466f756e6404c8204e6f20636f646520636f756c6420626520666f756e642061742074686520737570706c69656420636f646520686173682e2c4f75744f66426f756e6473042901204120627566666572206f757473696465206f662073616e64626f78206d656d6f7279207761732070617373656420746f206120636f6e7472616374204150492066756e6374696f6e2e384465636f64696e674661696c6564042d0120496e7075742070617373656420746f206120636f6e7472616374204150492066756e6374696f6e206661696c656420746f206465636f646520617320657870656374656420747970652e3c436f6e747261637454726170706564048c20436f6e7472616374207472617070656420647572696e6720657865637574696f6e2e1b24436f6e7472616374730140436f6e7472616374735772617070657214484d657461646174614f6654656d706c6174650101062c436f6465486173683c543e7854656d706c6174654d657461646174613c42616c616e63654f663c543e3e0084000324756e646566696e656400000000000000000000000000000000000000000004e02053746f726520746865206d6574612064657461696c73206f662074686520736d61727420657874656e73696f6e2074656d706c6174652e3054656d706c617465496e666f0101062c436f6465486173683c543e7454656d706c61746544657461696c733c42616c616e63654f663c543e3e00c40000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000004ec2053746f7265207468652064657461696c73206f66207468652074656d706c617465202845782d206f776e65722c2066726f7a656e20657463292e34457874656e73696f6e496e666f01010630543a3a4163636f756e74496484457874656e73696f6e417474726962757465733c42616c616e63654f663c543e3e0050000000000000000000000000000000000000000004882044657461696c73206f6620657874656e73696f6e2067657420757064617465642e38457874656e73696f6e4e6f6e636501000c75363420000000000000000008d4204e6f6e636520666f722074686520736d61727420657874656e73696f6e206163636f756e742069642067656e65726174696f6e2e5501205573696e67206578706c69636974206e6f6e636520617320696e206261746368207472616e73616374696f6e20616363
```

Comment:
is this supposed to be git ignored now?

As it is we'll always have this version included

Reply:
We don't need that. If we start using a lot of storage functions or rpcs in our mapping handlers, then to type them strongly we can add all the augment types as we do in SDK. But for now can be removed